lib/mllp/client.ex

```diff
@@ -193,24 +197,32 @@ defmodule MLLP.Client do
 
   MLLP.Client.start_link/4 will start a new MLLP.Client process.
 
-  This function will raise a `ArgumentError` if an invalid `ip_address()` is provided. 
+  This function will raise a `ArgumentError` if an invalid `ip_address()` is provided.
 
   ## Options
 
   * `:auto_reconnect_interval` - Specify the interval between connection attempts. Specifically, if an attempt
      to establish a connection fails, either post-init or at some point during the life span of the client, the value
      of this option shall determine how often to retry a reconnection. Defaults to 1000 milliseconds.
 
-  * `:reply_timeout` - Optionally specify a timeout value for receiving a response. Must be a positive integer or 
+  * `:use_backoff` - Specify if an exponential backoff should be used for connection. When an attempt
```

Comment:
I think `use_backoff` be the default. What do you think @vikas15bhardwaj ?

Reply:
yes, we can make it default